### ISSUE TITLE
Error in running ML.Dev.Pred.Category.Sig()

### ISSUE THREAD

**Issue by YihengJiang0912** [l-magnificence/Mime#46]
Dear authors, thank you very much for your powerful meta-ML tools! However, I got an error when running function ML.Dev.Pred.Category.Sig()

> res.ici <- ML.Dev.Pred.Category.Sig(train_data = input_list$training,
>                                     list_train_vali_Data = input_list,
>                                     candidate_genes = CAF_2_sig,
>                                      methods = c('nb','svmRadialWeights','rf','kknn','adaboost','LogitBoost'),
>                                      #methods = c('LogitBoost'),
>                                     seed = 5201314,
>                                   cores_for_parallel = 50)
> 
> ---loading the packages ---
> ---loading the function---
> [1] "There existing 25 genes in candidate genes"
> [1] "Intersetion of the candidate genes and the colnames of the provided data"
> [1] "There existing 25 genes in candidate genes, colnames of training data, colnames of validation data"
> [1] "nb"
> [1] "svmRadialWeights"
> [1] "kknn"
> [1] "rf"
> [1] "adaboost"
> [1] "LogitBoost"
> Error in (function (classes, fdef, mtable)  : 
>   unable to find an inherited method for function ‘pData’ for signature ‘"ExpressionSet"’
> In addition: There were 50 or more warnings (use warnings() to see the first 50)

I would be very grateful if you could give me some favor about that.
